flink-core/src/main/java/org/apache/flink/api/common/functions/util/RuntimeUDFContext.java

```diff
@@ -44,21 +43,24 @@
     private final HashMap<String, Object> initializedBroadcastVars = new HashMap<>();
```

Comment:
very very nit: Could we add a comment it is used only in the `CollectionExecutor` and can be dropped along with the `DataSet` API. It will help future readers.

Reply:
Sorry, I know realized I wasn't clear with my comment. I thought of putting that comment on the `RuntimeUDFContext` class... :facepalm: